### ISSUE TITLE
Complete a11y product health dashboard

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#56001]
## Description

Now that we're able to automate tracking issues thanks to @kylesoskin 's solution, it's appropriate timing to complete the a11y product health dashboard to:
- better equip CAIA to prioritize maintenance work between projects
- enable CAIA to better track existing a11y issues
- serve as a teams-facing tool to hold them accountable for their product's a11y health and spur action to improve

[Link to a11y dashboard](https://docs.google.com/spreadsheets/d/1lgzMiZZ-0rBPDuB2gRupBAjsyF0WTe005VvEHYrMxww/edit?usp=sharing)

## Tasks
- [x] Determine if there are missing products or teams from the list
- [x] Archive any products that are no longer in production
- [x] Document each product's most recent a11y audit
- [x] Using the product's label, automate its count of issues
- [x] If a product has open major issues that have not been fixed in 1-2 sprints, leave a comment asking why and track it within the dashboard
- [x] Label each product based on the below next step instructions

#### Audit ASAP
If a product has major issues and has not been reviewed in over 2 years.

#### Fix ASAP
If a product has a major issue that has not been resolved within 1-2 sprints post launch.

#### Blocked
If all remaining known issues are blocked.

#### Resolve issues
- If a product has major issues (0-2) remaining, but is still within a 1-2 spring post launch window
- If a product has non-major issues remaining (3-4)

#### Needs maintenance
If a product has no open issues and has not been reviewed in over a year

#### Needs AT testing
If a product has not been tested with assistive tech users yet

#### Perfect
If a product has no open issues, was audited within a year (or had a usability test with AT users within a year), meets WCAG 2.1 compliance, and has had mobile usability testing and assistive tech usability testing.

#### Needs team to fix
No team owns this product anymore

## Progress updates
[I've opened up a platform ticket to request for more features for the product directory to make this work a bit easier. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/56165)
